repl/scala-2.11/src/main/scala/org/apache/spark/repl/Main.scala

```diff
@@ -88,7 +88,8 @@ object Main extends Logging {
     }
 
     val builder = SparkSession.builder.config(conf)
-    if (SparkSession.hiveClassesArePresent) {
+    if (conf.getBoolean("spark.user.hive.catalog", true)
```

Comment:
why do we need this? isn't the enableHiveSupport the same as setting a config spark.sql.catalogImplementation